src/components/Avatar/Avatar.styles.ts

```diff
@@ -41,6 +41,12 @@ export const getClassNames = memoizeFunction((size: AvatarSize) =>
       border: `solid 2px ${palette.white}`,
       background: palette.white,
       color: palette.themeDark,
+      selectors: {
+        /* Remove when this is addressed: https://github.com/Microsoft/YamUI/issues/327 */
+        '> .y-icon': {
```

Comment:
maybe we should remove our uses of Context API and all the backflips in Jest unit tests and just use selectors like this...

Reply:
Good idea.  I'll try it.  It would be nice to back out of the context api stuff, it's been a bit painful.

Reply:
Context api is now gone.